--- a/src/anal/prvb.c
+++ b/src/anal/prvb.c
@@ -70,7 +70,7 @@
 				if( (dialect_of(&WorkGkword) & PROSE) ) 
 					continue;
 					
-				strcpy(stem_of(&WorkGkword),stem_of(&WorkGkword)+1);
+				memmove(stem_of(&WorkGkword),stem_of(&WorkGkword)+1,strlen(stem_of(&WorkGkword)+1)+1);
 				add_morphflag(morphflags_of(prvb_gstr_of(&WorkGkword)),DOUBLED_CONS);
 				add_dialect(stem_gstr_of(&WorkGkword),(EPIC|POETIC));
 			}
